internal/service/searchindex/resource_search_index_test.go

```diff
@@ -23,30 +23,30 @@ const (
 
 func TestAccSearchIndexRS_basic(t *testing.T) {
 	var (
-		projectID                                        = os.Getenv("MONGODB_ATLAS_PROJECT_ID")
-		clusterName, clusterNameStr, clusterTerraformStr = acc.GetClusterInfo(projectID)
-		indexName                                        = acctest.RandomWithPrefix("test-acc-index")
-		databaseName                                     = acctest.RandomWithPrefix("test-acc-db")
+		orgID                                                          = os.Getenv("MONGODB_ATLAS_ORG_ID")
+		projectIDStr, clusterName, clusterNameStr, clusterTerraformStr = acc.GetClusterInfo(orgID)
```

Comment:
would it be useful to return also the projectID (like clusterName) to check it in test?

Reply:
We cannot do that for the case of project id because it is a computed attribute (obtained once config is actually applied). It is similar to acceptance tests that create projects as part of the config and cannot check the actual value of the project id.